Package.swift

```diff
@@ -15,18 +15,22 @@ import PackageDescription
 let package = Package(
   name: "swift-numerics",
   products: [
+    .library(name: "BigIntModule", targets: ["BigIntModule"]),
     .library(name: "ComplexModule", targets: ["ComplexModule"]),
     .library(name: "Numerics", targets: ["Numerics"]),
     .library(name: "RealModule", targets: ["RealModule"]),
   ],
   dependencies: [
+    .package(url: "https://github.com/attaswift/BigInt.git", from: "3.0.0")
   ],
   targets: [
+    .target(name: "BigIntModule", dependencies: []),
     .target(name: "ComplexModule", dependencies: ["RealModule"]),
     .target(name: "Numerics", dependencies: ["ComplexModule", "RealModule"]),
```

Comment:
```suggestion
    .target(name: "Numerics", dependencies: ["BigIntModule", "ComplexModule", "RealModule"]),
```

Also:
* This file `Copyright (c) 2019 - 2020`
* Other files `Copyright (c) 2020`

Reply:
I think the `Numerics` target still needs the `BigIntModule` dependency (see above).